kv/txn.go

```diff
@@ -48,8 +47,6 @@ func RunInNewTxn(store Storage, retryable bool, f func(txn Transaction) error) e
 
 		err = f(txn)
 		if err != nil {
-			err1 := txn.Rollback()
```

Comment:
Is it safe to remove this?

Reply:
I have the same question.

Reply:
I don't think this change is appropriate. We should keep the `txn.Rollback()` and just check if the error is kv.ErrInvalidTxn is better.